score/mw/com/impl/configuration/config_parser.cpp

```diff
@@ -725,14 +874,27 @@ auto ParseServiceInstances(const score::json::Any& json, TracingConfiguration& t
 // coverity[autosar_cpp14_a15_5_3_violation]
 auto ParseLolaEventTypeDeployments(const score::json::Any& json, LolaServiceTypeDeployment& service) noexcept -> bool
 {
-    const auto& events = json.As<score::json::Object>().value().get().find(kEventsKey.data());
-    if (events == json.As<score::json::Object>().value().get().cend())
+    auto json_obj = json.As<score::json::Object>();
+    if (!json_obj.has_value()) {
+        return false;
```

Comment:
Behavioral change. Should terminate.

```suggestion
        score::mw::log::LogFatal("lola") << "LolaEventTypeDeployments not parsable.";
        SCORE_LANGUAGE_FUTURECPP_ASSERT_PRD(false);
```